torch/nn/modules/instancenorm.py

```diff
@@ -11,22 +13,27 @@ def __init__(self, num_features, eps=1e-5, momentum=0.1, affine=False,
     def _check_input_dim(self, input):
         return NotImplemented
 
-    def _load_state_dict_key_mismatch(self, full_name, name, is_missing):
-        if not is_missing and not self.track_running_stats and \
-                name in ('running_mean', 'running_var'):
-            raise KeyError(
-                'Unexpected running stats buffer "{name}" in state_dict for '
-                '{klass} with track_running_stats=False. If you are trying to '
-                'load a checkpoint saved before 0.4.0, this may be expected '
-                'because {klass} does not track running stats by default '
-                'anymore since 0.4.0. If running stats are not used, remove '
-                'them from state_dict before calling load_state_dict. '
-                'Otherwise, set track_running_stats=True in {klass} to use '
-                'running stats. See the documentation of {klass} for more '
-                'details.'
-                .format(name=full_name, klass=self.__class__.__name__))
-        super(_InstanceNorm, self)._load_state_dict_key_mismatch(
-            full_name, name, is_missing)
+    def load_local_state_dict(self, local_state_dict, version, strict):
+        # at version 0: removed running_mean and running_var when
```

Comment:
Can we make this 1? Otherwise it means that previously all modules had version -1, which is a bit counterintuitive.